src/jdk.internal.jvmstat/share/classes/sun/jvmstat/perfdata/monitor/protocol/local/LocalVmManager.java

```diff
@@ -76,10 +76,9 @@ public LocalVmManager() {
      */
     public LocalVmManager(String user) {
         this.userName = user;
-
+        setTempDirs();
```

Comment:
Is this call necessary? It looks like we don't need to hold `tmpdirs` anymore. Can we remove `tmpdirs` from class member and can call `PerfDataFile.getTempDirectories()` directly at L138?

Reply:
Hi - right, it's not necessary, I almost took it out, but had a minor concern about leaving tmpdirs unset.
But removing tmpdirs as a member is fine also.  That is testing well, will push an update shortly.